File: 프로그래머스/3/12987. 숫자 게임/숫자 게임.js
function solution(A, B) {
    A.sort((a, b) => b - a);
    B.sort((a, b) => b - a);
    let b = 0;
    let answer = 0;
    if (A[A.length - 1] >= B[0]) return 0;
    for (let i = 0; i < A.length; i++) {
        if (A[i] < B[b]) {
            b++; answer++;
        }
    }
    return answer;
}

File: 프로그래머스/3/12987. 숫자 게임/숫자 게임.py
from collections import deque
def solution(A, B):
    A.sort(reverse=True)
    B.sort(reverse=True)
    B = deque(B)
    if A[-1] >= B[0]: return 0
    answer = 0
    a = 0
    while len(B):
        if A[a] < B[0]:
            answer += 1
            B.popleft()
        elif A[a] >= B[0]:
            B.pop()
        a += 1
    return answer